api/v1alpha1/gatekeeper_types.go

```diff
@@ -189,38 +202,24 @@ type WebhookConfig struct {
 	//nolint:lll
 	AdmissionEventsInvolvedNamespace *Mode `json:"admissionEventsInvolvedNamespace,omitempty"`
 
-	// +optional
-	// +kubebuilder:validation:Enum:=Ignore;Fail
-	FailurePolicy *admregv1.FailurePolicyType `json:"failurePolicy,omitempty"`
-
-	// NamespaceSelector is a label selector to define which namespaces should be handled by the
-	// admission webhook.
-	//
-	// +optional
-	NamespaceSelector *metav1.LabelSelector `json:"namespaceSelector,omitempty"`
-
-	// Operations specifies a list of API operations to be checked by the admission webhook. The
-	// default value is ["CREATE","UPDATE"].
-	// See https://open-policy-agent.github.io/gatekeeper/website/docs/customize-admission/#enable-validation-of-delete-operations.
-	//
-	// +optional
-	//nolint:lll
-	Operations []OperationType `json:"operations,omitempty"`
-
 	// DisabledBuiltins is a list of specific OPA built-in functions to disable. By default, http.send
 	// is disabled.
 	// See https://open-policy-agent.github.io/gatekeeper/website/docs/customize-startup#disable-opa-built-in-functions.
 	//
 	// +optional
 	DisabledBuiltins []string `json:"disabledBuiltins,omitempty"`
 
+	// Deprecated: spec.webhook.logMutations is deprecated. Use the newer field at
+	// spec.mutatingWebhookConfig.logMutations, which will override what is set here.
```

Comment:
(nit) New is implied if this is deprecated.

```suggestion
	// Deprecated: spec.webhook.logMutations is deprecated. Use the field at
	// spec.mutatingWebhookConfig.logMutations, which will override what is set here.
```